src/qat/purr/compiler/instructions.py

```diff
@@ -869,9 +869,7 @@ def instructions(self) -> List[QuantumInstruction]:
         instructions = [Synchronize(list(self._entangled_qubits))]
```

Comment:
Is it also worth changing this to targets? It seems to be part of the confusion

Reply:
This will expand to all `Qubit`s measured in the `MeasureBlock`. Does not care about entanglement.

Reply:
I ment measurement targets not the entangled_qubits! So the conclusions is to change this to measurement targets from chats.

Reply:
Done as of last commit.